include/Scrape.php

```diff
@@ -235,7 +235,9 @@ function scrape_feed($url) {
 	$a = get_app();
 
 	$ret = array();
-	$s = fetch_url($url);
+	$cookiejar = tempnam('/tmp', 'cookiejar-scrape-feed-');
```

Comment:
Ah, didn't know about that function, thanks!  Will change it and wait for some more articles to come through, then I'll resubmit the pull request.